packages/angular_devkit/schematics/src/formats/path.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { normalize, schema } from '@angular-devkit/core';
+
+
+export const pathFormat: schema.SchemaFormat = {
+  name: 'path',
+  formatter: {
+    async: false,
+    validate: (path: string) => {
```

Comment:
We should normalize the path instead of verifying if it's normalized already.

Reply:
We can't do that in a format though. It just verifies that the data is following a format, it can't modify it.